src/vector/index.js

```diff
@@ -30,7 +30,6 @@ require('babel-polyfill');
 
 // CSS requires: just putting them here for now as CSS is going to be
```

Comment:
This was pulling in all the CSS: components.css is the catted-together bundle of all the components' CSS and the others were various other ones we used. Requiring them here pulled them in to webpack's build process which pulled them through the extract text plugin, eventually getting the end CSS bundle out the other end.

The comment was me putting the CSS requires in the vector index file rather than working out if there was a better place to put them, as at the time there was a refactor on the horizon of how vector's CSS worked (although that never came to pass).